airflow/config_templates/config.yml

```diff
@@ -394,6 +394,15 @@
       type: boolean
       example: ~
       default: "True"
+    - name: lazy_discover_providers
```

Comment:
I added  a new config for that -> I think it makes sense to keep it separate from plugins